### DISCUSSION TITLE
part-ii/2005-64/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#222] [Question comments]
Part II 2005 Graph Theory: 1.II .17F 

https://questions.tripos.org/part-ii/2005-64/

**Comment by walrus1729** [comment]
<details>
<summary>Solution</summary>

The first part is bookwork. If a connected graph of order is not acyclic, then we can remove edges from cycles until we have an acyclic connected graph with fewer edges, a contradiction.

For the next part, we induct on $t$, the base case $t=0$ being trivial (every graph with minimum degree 1 has a vertex). For the inductive step, we can remove a leaf vertex (vertex of degree one) from a tree, find it as a subgraph of $G$, then add the leaf vertex back because each vertex in our subgraph will have a neighbour we haven't used in our tree of order $t - 1$ since $\delta(G) \geq t - 1$. 

For the last part, one solution is to take $T$ to be the star graph, which has $\delta(G) = t - 1$, and will certainly not be a subgraph of a $(t-2)$-regular graph. Alternatively, observe that $\delta(K_{t-1}) = t - 2$, but $|K_{t-1}| < |T|$, so $T$ cannot be a subgraph of $K_{t-1}$.

</details>
